backend/core/tasks.py

```
@@ -0,0 +1,41 @@
+            if owner.email not in owner_controls:
+                owner_controls[owner.email] = []
+            owner_controls[owner.email].append(control)
+
+    # Send personalized email to each owner
+    for owner_email, controls in owner_controls.items():
+        send_notification_email(owner_email, controls)
+
+
+@task()
+def send_notification_email(owner_email, controls):
+    subject = f"You have {len(controls)} expired controls"
```

Ruff (0.8.2)

Local variable `subject` is assigned to but never used

Remove assignment to unused variable `subject`

(F841)